Library/ViewModels/EditorialProjectsViewModel.swift

```diff
@@ -112,11 +112,17 @@ private func editorialImageName(for tagId: DiscoveryParams.TagID) -> String {
       return "go-rewardless-modal-large"
     }
     return "go-rewardless-modal"
+  case .lightsOn:
+    return "go-rewardless-modal"
```

Comment:
Seeing as we're removing go rewardless we probably don't need this function (unless we need a different image for iPad), this seems to return the image name for go rewardless though?

Reply:
I'll address that in the next PR for the editorial list. I'm not including the asset with this PR, it's in the next.